docs/architecture/components.mdx

```diff
@@ -92,57 +94,60 @@ The Node Collector is also responsible for recording some k8s metrics, and colle
 
 **Type**: The instrumentor is a Kubernetes operator
 
-**K8s Resource**: Deployment / `odigos-instrumentor`
+**K8s Resource**: `kubectl get deployments -n odigos-system odigos-instrumentor`
 
-**Role**: The instrumentor is responsible for managing the auto instrumentation agents which collect the telemetry data from the selected sources.
+**Role**: The instrumentor is responsible for managing the auto instrumentation agents which collect the telemetry data from the Odigos sources.
 
 - Create and manage the `instrumentationconfig.odigos.io` objects
-  - Triggers the `odiglet` to inspect runtime details of the workload.
+  - Triggers `odiglet` to inspect runtime information of the workload.
   - Triggers the scheduler to deploy the Odigos pipeline.
-  - Merge the Odigos instrumentation rules and odigos config to a per-workload configuration to be used by the agent.
+  - Merge the Odigos instrumentation rules and odigos config to a workload scope configuration to be used by the agent.
 - Inject the instrumentation device into the workload manifest resource as virtual device.
```

Comment:
Do we have any docs on what an instrumentation device is?

Reply:
I don't think so, but since we are moving away from devices and will deprecate them in the near future, we can skip this IMO

Reply:
sgtm